docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,10 +367,17 @@ which is a CDI bean and can be ``@Inject``ed.
 
 In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+[IMPORTANT]
+====
+Using a database to store transaction logs has several benefits, but in comparison with using the file system to store the logs, you might notice a reduction in performance.
```

Comment:
s/benefits, but in comparison/benefits but, in comparison/

Reply:
If you are going to recommend one over the other you need delineate the pros and cons of each store.